webpack/assets/javascripts/react_app/components/Editor/Editor.js

```diff
@@ -110,106 +110,108 @@ class Editor extends React.Component {
     };
 
     return (
-      <div id="editor-container">
-        <ToastNotification
-          id="preview_error_toast"
-          type="error"
-          className={showError ? '' : 'hidden'}
-          onDismiss={() => dismissErrorToast()}
-        >
-          {errorText}
-        </ToastNotification>
-        <EditorNavbar
-          changeDiffViewType={changeDiffViewType}
-          changeTab={changeTab}
-          changeSetting={changeSetting}
-          modes={EDITOR_MODES}
-          themes={EDITOR_THEMES}
-          keyBindings={EDITOR_KEYBINDINGS}
-          mode={isRendering ? 'Text' : mode}
-          theme={theme}
-          keyBinding={keyBinding}
-          value={value}
-          renderedEditorValue={renderedEditorValue}
-          diffViewType={diffViewType}
-          template={template}
-          selectedView={selectedView}
-          isDiff={template ? value !== template : false}
-          isMasked={isMasked}
-          isRendering={isRendering}
-          isLoading={isLoading}
-          isFetchingHosts={isFetchingHosts}
-          isSearchingHosts={isSearchingHosts}
-          importFile={importFile}
-          showImport={showImport}
-          showPreview={showPreview}
-          showHostSelector={showHostSelector}
-          showHide={showHide}
-          revertChanges={revertChanges}
-          previewTemplate={previewTemplate}
-          hosts={hosts}
-          filteredHosts={filteredHosts}
-          selectedHost={selectedHost}
-          renderPath={renderPath}
-          toggleMaskValue={toggleMaskValue}
-          toggleRenderView={toggleRenderView}
-          toggleModal={toggleModal}
-          previewResult={previewResult}
-          searchQuery={searchQuery}
-          onHostSelectToggle={onHostSelectToggle}
-          onHostSearch={onHostSearch}
-          onSearchClear={onSearchClear}
-          isSelectOpen={isSelectOpen}
-          showError={showError}
-          fetchAndPreview={fetchAndPreview}
-        />
-        <EditorView
-          {...editorViewProps}
-          key="editorPreview"
-          name={editorNameTab.preview}
-          isSelected={selectedView === 'preview'}
-          className="ace_editor_form ace_preview"
-        />
-        <EditorView
-          {...editorViewProps}
-          key="editorCode"
-          name={editorNameTab.input}
-          isSelected={selectedView === 'input'}
-          className="ace_editor_form ace_input"
-        />
-        <div
-          id="diff-table"
-          className={selectedView === 'diff' ? '' : 'hidden'}
-        >
-          <DiffView
-            oldText={template || ''}
-            newText={value}
-            viewType={diffViewType}
+      <div id="react-ace">
```

Comment:
this div isn't related to styles but was created for mounting and for testing, so basically we don't need it at all, we can move the related tests to use the div beneath `editor-container`

for example: 
```ruby
    # test/integration/ptable_js_test.rb
    test "edit page" do  
      fill_in_editor_field('#editor-container', editor_text) 
      assert has_editor_display?('#editor-container, editor_text')
    end
```